### ISSUE TITLE
Oneway attribute should be considered an optional flag in network link XML elements

### ISSUE THREAD

**Issue by mfitz** [arup-group/genet#55]
GeNet's network simplification assumes that all MATSim network links have the `oneway` attribute flag, and throws an error when this is not the case. For example, when simplifying Matesto's test town network:

```
...
2021-02-12 14:26:37,796 - Processing links for all paths to be simplified
2021-02-12 14:26:37,804 - Adding new simplified links
2021-02-12 14:26:37,862 - Generated 0 link ids.
2021-02-12 14:26:37,873 - Added 6 links
2021-02-12 14:26:37,876 - Simplified graph: 14 to 8 nodes, 26 to 14 edges
2021-02-12 14:26:37,876 - Updating the Schedule
2021-02-12 14:26:37,877 - Updated Stop Link Reference Ids
2021-02-12 14:26:37,878 - Updated Network Routes
2021-02-12 14:26:37,878 - Simplification resulted in 18 links being simplified.
2021-02-12 14:26:37,909 - Writing working-dir/genet/network.xml
Traceback (most recent call last):
  File "simplify_network.py", line 68, in <module>
    n.write_to_matsim(output_dir)
  File "/genet/core.py", line 1248, in write_to_matsim
    matsim_xml_writer.write_matsim_network(output_dir, self)
  File "/genet/outputs_handler/matsim_xml_writer.py", line 84, in write_matsim_network
    link_attributes = prepare_link_attributes(deepcopy(link_attribs))
  File "/genet/outputs_handler/matsim_xml_writer.py", line 58, in prepare_link_attributes
    validate_link_data(link_attributes)
  File "/genet/validate/network_validation.py", line 8, in validate_link_data
    raise AttributeError('Attribute: {} missing from link: {}'.format(necessary_link_attrib, link_attributes))
AttributeError: Attribute: oneway missing from link: {'id': '1-3', 'modes': {'bus', 'cycle', 'car'}, 'permlanes': 1.0, 'capacity': 1000.0, 'freespeed': 10.0, 'to': '3', 'from': '1', 'length': 500.0}
ERROR: Step 'genet-testtown12' failed ('1') !
```

Where the `oneway` attribute is present, it should be preserved in the simplified network, but GeNet should not raise an error when it is missing in the source link.

To reproduce the above effect, use the following GeNet script and parameters:

```
    "simplify_network.py",
    "-n", "matsim12-test-town/network.xml",
    "-s", "matsim12-test-town/transitschedule.xml",
    "-p", "epsg:27700",
    "-od", "working-dir/genet"
```

MATSim's [network DTD](https://www.matsim.org/files/dtd/network_v2.dtd) describes the `oneway` attribute thus:

```xml
<!ELEMENT link (attributes?)>
<!ATTLIST link id          CDATA #REQUIRED
               from        CDATA #REQUIRED
               to          CDATA #REQUIRED
               length      CDATA #REQUIRED
               freespeed   CDATA #REQUIRED
               capacity    CDATA #REQUIRED
               permlanes   CDATA #REQUIRED
               oneway      CDATA #FIXED    "1"
               origid      CDATA #IMPLIED
               type        CDATA #IMPLIED
               modes       CDATA "car">
```

**Comment by KasiaKoz** [comment]
Indeed, it is not a required atrribute for a matsim network https://www.matsim.org/files/dtd/network_v2.dtd

```
               oneway      CDATA #FIXED    "1"
```

**Comment by mfitz** [comment]
Fixed by [commit 0ef69a8896da02ff08993f2748e5fd3e92eff124](https://github.com/arup-group/genet/commit/0ef69a8896da02ff08993f2748e5fd3e92eff124)
